docs/reference/security/threat-model.md

```diff
@@ -0,0 +1,373 @@
+# Knative Threat Model
+
+This document describes the Knative threat model. When vulnerabilities are
+reported to the project, we consult this document to determine whether the
+report describes a potential exploit, and if so to determine the severity of the
+exploit. As we develop new features, we consult this document to consider their
+impact on the threat model. Note that this threat model covers Serving,
+Eventing; some sections and threats may only apply to certain components of the
+project. As Knative Functions largely executes at build time on the application
+developer's infrastructure, it needs a different threat model more focused on
+supply chain security threats (which it largely inherits from
+[CNCF Buildpacks](https://buildpacks.io/)).
+
+Knative builds on the capabilities of the Kubernetes cluster, and exposes both
```

Comment:
This second paragraph seem to frame the Knative threat model and its relationship with best practices  - again I do not think it is a helpful starting statement for a page spelling out what is the Knative threat model - the information should be in the doc, but we first need to layout what is the Knative threat model...